src/utils/image_server.cljs

```diff
@@ -58,3 +60,55 @@
        "&clock="
        clock
        "&addRing=1"))
+
+(defn get-account-qr-image-uri
+  [{:keys [key-uid public-key port qr-size]}]
+  (let [profile-qr-url         (str status-profile-base-url public-key)
+        base-64-qr-url         (js/btoa profile-qr-url)
+        profile-image-type     "large"
+        error-correction-level 4
```

Comment:
Here could you use `(get-error-correction-index-for-level :highest)` instead of the magic number?

Reply:
addressed here : https://github.com/status-im/status-mobile/pull/15755/commits/ddbe0e089796e1dead6cddb88adda90dc4acd8ba